src/main/java/duke/Gui.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+/**
+ * A GUI for Duke using FXML.
```

Comment:
I like this part very much, it is very appreciable to write the start method outside the original Duke file, which minimizes the influence of building the front end to the original code.